tests/test_register.py

```diff
@@ -87,7 +89,8 @@ def test_non_existent_package(register_settings):
 
 
 def test_values_from_env(monkeypatch):
-    """Test calling main via cli"""
+    """Package registration should behave as expected
```

Comment:
Same thing here. Further, the first line should fit on one-line, including a period and additional information should be specified after a blank line in paragraph form.